content/en/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -14,7 +14,7 @@ Init containers can contain utilities or setup scripts not present in an app ima
 You can specify init containers in the Pod specification alongside the `containers`
 array (which describes app containers).
 
-In Kubernetes, a [sidecar container](/docs/concepts/workloads/pods/sidecar-containers/) is a container that
+In Kubernetes, a [init container](/docs/concepts/workloads/pods/init-containers/) is a container that
```

Comment:
This is not correct. An basic init container does not continue to run. It is more accurate calling it a sidecar container, even though it uses the `initContainers` section of the pod spec with the special `restartPolicy` to make it become a sidecar.